packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -1111,6 +1111,7 @@ describe('RichTextEditorMentionListbox', () => {
             expect(pageObject.isMentionListboxOpened()).toBeFalse();
         });
 
+        /* Disabled due to intermittency. See: https://ni.visualstudio.com/DevCentral/_workitems/edit/2632606
         it('should commit mention into the editor on Enter', async () => {
```

Comment:
Disable tests using `xit` instead of having it commented out.

Can clarify the intermittent docs on disabled tests if needed: https://github.com/ni/nimble/blob/adde25d1cad4b59868590e81684a622bbd38b23e/CONTRIBUTING.md#handling-intermittent-test-failures